data/json/npcs/talk_tags.json

```
@@ -941,18 +1102,37 @@
     "text": [
       "Tell me about how you survived the cataclysm.",
       "How did you survive the cataclysm?",
-      "What was the cataclysm like for you?"
+      "What was the cataclysm like for you?",
+      "How did you make it through the initial chaos?",
+      "Tell me how you survived the initial wave of the Cataclysm.",
+      "Was it rough surviving thus far?"
```

```suggestion
      "Was it rough surviving this far?"
```
"Thus" is a bit too archaic for you to just randomly drop. Context: This is what the player says when asking NPCs about their stories.

---

"Thus" may be a bit archaic, but we have medieval knaves, people from all sorts of life, professors in the game. Something that is a bit odd but used in modern times doesn't seem like a mad fever dream.